### DISCUSSION TITLE
postgres data gets deleted when i restart server

### DISCUSSION THREAD

**Discussion by feranzie** [docker-library/postgres#1156] [General]
Docker version 20.10.22, build 3a2c30b
windows 10

every time I ingest data it works and I'm able to use the data in pgcli, but once I restart the server its no longer there

`winpty docker run -it \
  -e POSTGRES_USER="root" \
  -e POSTGRES_PASSWORD="root" \
  -e POSTGRES_DB="ny_taxi" \
  -v /c/Users/DELL/Desktop/data-engineering-zoomcamp/week_1_basics_n_setup/2_docker_sql/ny_taxi_postgres_data:/var/lib/postgresql/data/: \
  -p 5432:5432 \
  postgres:13`

**Comment by LaurentGoderre** [comment]
That's strange. Restarting doesn't delete files in the volume on the host. Is it possible there is script deleting the content of the folder n startup?

**Comment by tianon** [comment]
:eyes: `/c/Users/DELL/Desktop/data-engineering-zoomcamp/week_1_basics_n_setup/2_docker_sql/ny_taxi_postgres_data`

That looks like you're probably using MinGW on Windows, perhaps via Git Bash?

If you convert that to `'C:\Users\DELL\Desktop\data-engineering-zoomcamp\week_1_basics_n_setup\2_docker_sql\ny_taxi_postgres_data'`, you might have better luck (Docker Desktop should then be able to convert/share it correctly).
